geo/geo

```diff
@@ -68,7 +68,7 @@ version() {
 
 # Fetches WAN IP address
 wan_search() {
-  httpGet https://api.ipify.org?format=json | grep -Eo "[0-9.]*"
+  httpGet https://api.ipify.org?format=json | grep -Eo "[0-9.]+"
```

Comment:
If i remember correctly ```+``` does not work on linux, does ```*``` not work for mac? If thats the case a lot more tools are broken and they will all have to be wrapped with a platform conditional or do something similar to the http_get logic for that character.

Reply:
When I run these lines on my system (Fedora Linux)

```
http GET https://api.ipify.org?format=json | grep -Eo "[0-9.]*"
http GET https://api.ipify.org?format=json | grep -Eo "[0-9.]+"
```

I get the same output (an IP address) from both statements.

Reply:
So it has been a while since I wrote these, but I know I have always used *, I recall it having to do with the plus not working in my terminal. Currently the Ubnutu subsystem for Windows agrees with you. Does the * not work on mac?

Reply:
`*` does not works. if you think changing to `+` will break on some environments we could use something like:

```bash
if [ "$(uname)" = "Darwin" ]; then
   WILDCHAR="+"
 else
   WILDCHAR="*"
fi
```